### ISSUE TITLE
Improve the admin page

### ISSUE THREAD

**Issue by phoenixxxx** [rishigupta9999/Newton#21]
- [x]  See the list of parties.
- [x]  See the list of questions.
- [x]  See the list of users.
  - [x] See the last time the user logged in. (decide when to nudge them)
- [x] See the user submitted feedback.
  (https://github.com/rishigupta9999/Newton/issues/45)
- When you select a party
  - [x] You can see all the data we have stored and generate on the party (can be a raw text file that can be manually edited)
  - [x] You can see all the users of the party.
  - [x] You can send and email to a party.
- You can select a user
  - [x] You can see all the data stored and generated on the user 
    - [x] Questions answered
    - [x] Feedback they posted (from the /feedback page)
    - [x] Data from Spotify.
    - [x] Data from Facebook.
    - [x] Data we have extrapolated and manually added, all the parties the user is a member of etc... this can simply be a raw text file as well.)
  - [x] You can send an email to a member.
    (this should be as easy as calling Outreach.direct_user_message(user.email, header, message, footer).deliver_now once it is merged into master).

This will be a continuation of the work at:
http://localhost:3000/administrator
